classes/image/imagemagick.php

```diff
@@ -158,6 +158,14 @@ protected function _mask($maskimage)
 		$this->exec('convert', $command);
 	}
 
+	protected function _blur($radius, $sigma)
+	{
+		extract(parent::_blur($radius, $sigma));
+
+		$image = '"'.$this->image_temp.'"';
+		$this->exec('convert', $image.' -blur '.$radius.'x'.$sigma.' '.$image);
```

Comment:
All other parameters are expected to be integers, harm can be done with strings, but not by automatic toString format. At least use number_format to get spaces and commas right.